src/Control/Monad/Rec/Class.purs

```diff
@@ -93,6 +97,14 @@ tailRec f = go <<< f
   go (Loop a) = go (f a)
   go (Done b) = b
 
+-- | Create a pure tail-recursive function of two arguments
```

Comment:
Could you add docs that refer to `loop2` here? Otherwise, it's not immediately clear that one would know to use `loop2` in tandem with `tailRec2`